README.md

```diff
@@ -6,42 +6,69 @@
 
 # ssi-service
 
-## Introduction
+HTTP-API driven web service that wraps our [ssi-sdk](https://github.com/TBD54566975/ssi-sdk).
 
+## Introduction
 The Self Sovereign Identity Service (SSIS) facilitates all things relating to [DIDs](https://www.w3.org/TR/did-core/)
-and [Verifiable Credentials](https://www.w3.org/TR/vc-data-model) -- in a box! The service is a part of a larger
+and [Verifiable Credentials](https://www.w3.org/TR/vc-data-model) - in a box! The service is a part of a larger
 Decentralized Web Platform architecture which you can learn more about in our
-[collaboration repo](https://github.com/TBD54566975/collaboration). The SSI Service is a HTTP-API driven web service
-that wraps the [ssi-sdk](https://github.com/TBD54566975/ssi-sdk). The core functionality of the SSIS includes,
-but is not limited to: interacting with the standards around Verifiable Credentials, Credential Revocations, requesting
-Credentials, exchanging Credentials, data schemas for Credentials and other verifiable data, messaging using
-Decentralized Web Nodes, and usage of Decentralized Identifiers. Using these core standards, the SSIS enables robust
-functionality to facilitate all verifiable interactions such as creating, signing, issuing, curating, requesting,
-revoking, exchanging, validating, verifying credentials in varying degrees of complexity.
-
-For more information, see the [vision document](doc/VISION.md).
-
-![ssi-service](doc/ssi-service.png)
+[collaboration repo](https://github.com/TBD54566975/collaboration).
+
+## Core Functionality
+- Interacting with the standards around Verifiable Credentials
+- Credential Revocations
+- Requesting Credentials
+- Exchanging Credentials
+- Data Schemas (for credentials and other verifiable data)
+- Messaging using Decentralized Web Nodes (DWNs)
+- Usage of Decentralized Identifiers
+
+### Verifiable Interactions <br />
+- Create
+- Sign
+- Issue
+- Curate
+- Request
+- Revoke
+- Exchange
+- Validate
+- Verify
+
+## Use Cases (more to come!)
+### Business: Issuing Verifiable Credentials <br />
+[Follow Tutorial](https://developer.tbd.website/docs/tutorials/issue-verifiable-credential-manually)
+
+Steps to issue an Employment Status Credential:
+1. Spin up and host the SSI-Service
+2. Add the ability for your employees to click 'apply for a credential' on your internal EMS (should we show a front end button code example)
+3. [Create an Issuer DID](blob/main/integration/common.go#L36) for your business
+4. [Create a Schema](blob/main/integration/common.go#L66)
+5. [Create a Credential Manifest](blob/main/integration/common.go#L154)
+6. [Submit a Credential Application](blob/main/integration/common.go#L173)
+7. Allow the ability for your employees to [revoke their credential](where?)
+
+'ADD IMAGE OF THIS FLOW THAT NEAL MADE'
```

Comment:
should we add this image before shipping it? 

Let me know if I need to spice it up / if you need me to find it again!

Reply:
Are you saying to wait till changes in this repo are shipped? or before just shipping the readme changes?

Reply:
@nitro-neal ?

Reply:
Where can folks revoke, what function should I point ppl to? @decentralgabe @nitro-neal @mistermoe ?

`Allow the ability for your employees to [revoke their credential](where?)`

Reply:
@decentralgabe @nitro-neal checking again